### PR TITLE
mediatomb/gerbera: Add release note information for 21.03

### PR DESCRIPTION
**Pull request by ardumont**
Building the documentation [1] does not complain about anything

[1] https://nixos.org/manual/nixpkgs/stable/#chap-contributing

Related to #93450#issuecomment-706962271

### CODE REVIEW

**Comment by ardumont** [comment]
Updated and fixed the conflict

**Comment by SuperSandro2000** [comment]
@ardumont please resolve the merge conflict.

**Comment by ardumont** [comment]
> @ardumont please resolve the merge conflict.

Btw, I've rebased and fixed the conflict again (I did also last time you asked...). 
t'd be nice if that could land at some point...

(It was also initially from another PR and it got split... The actual code describing
this landed but the doc never made it into 20.09. tl; dr, I also had to rebase and
resolve conflicts time and again before, so it's getting a bit repetitive/tedious...)

Thanks for your time and consideration.

Cheers,

**Comment by schmittlauch** [comment]
This obviously still needs to go into 21.05, because the changes are already merged but the release note is not. Somehow the manual is failing to build, can you check and resolve this @ardumont?

/cc @jonringer

**Comment by schmittlauch** [comment]
@ardumont Did you see that unfortunately the NixOS tests for mediatomb are failing because of gerbera, while mediatomb still succeeds?

First, the `mkdir` path for the gerbera test is wrong. But even after changing `mediatomb` for `gerbera` in there, the `curl` test fails:

```serverGerbera: must succeed: mkdir -p /var/lib/gerbera/{pictures,audio}
serverGerbera: waiting for the VM to finish booting
serverGerbera: connected to guest root shell
serverGerbera: (connecting took 0.00 seconds)
(0.00 seconds)
(0.02 seconds)
serverGerbera: must succeed: chown -R mediatomb:mediatomb /var/lib/gerbera
(0.03 seconds)
serverGerbera: must succeed: sleep 2
(2.02 seconds)
serverGerbera: waiting for success: systemctl restart mediatomb
serverGerbera # [   27.260642] gerbera[812]: 2021-05-28 00:02:00   info: Gerbera shutting down. Please wait...
serverGerbera # [   27.264609] systemd[1]: Stopping Gerbera media Server...
serverGerbera # [   27.365058] gerbera[812]: 2021-05-28 00:02:00   info: Gerbera exiting. Have a nice day.
serverGerbera # [   27.373946] systemd[1]: mediatomb.service: Succeeded.
serverGerbera # [   27.376390] systemd[1]: Stopped Gerbera media Server.
serverGerbera # [   27.378154] systemd[1]: mediatomb.service: Consumed 240ms CPU time, received 11.6K IP traffic, sent 11.6K IP traffic.
serverGerbera # [   27.387464] systemd[1]: Started Gerbera media Server.
(0.16 seconds)
serverGerbera # [   27.437781] gerbera[918]: 2021-05-28 00:02:00   info: Gerbera UPnP Server 1.8.1 - http://gerbera.io/
serverGerbera # [   27.440490] gerbera[918]: 2021-05-28 00:02:00   info: Copyright 2016-2021 Gerbera Contributors.
serverGerbera # [   27.443417] gerbera[918]: 2021-05-28 00:02:00   info: Gerbera is free software, covered by the GNU General Public License version 2
serverGerbera # [   27.448795] gerbera[918]: 2021-05-28 00:02:00   info: Loading configuration from: /nix/store/j59xhwzg6zysgc90nwi9s8knyrjiqz28-config.xml
serverGerbera # [   27.452741] gerbera[918]: 2021-05-28 00:02:00   info: Checking configuration...
serverGerbera # [   27.455164] gerbera[918]: 2021-05-28 00:02:00   info: Configuration check succeeded.
serverGerbera # [   27.462059] gerbera[918]: 2021-05-28 00:02:00   info: About to migrate metadata from mt_cds_object to mt_metadata
serverGerbera # [   27.464710] gerbera[918]: 2021-05-28 00:02:00   info: No data will be removed from mt_cds_object
serverGerbera # [   27.477374] gerbera[918]: 2021-05-28 00:02:00   info: Migrated metadata - object count: 0
serverGerbera # [   27.486056] gerbera[918]: 2021-05-28 00:02:00   info: Loading 0 configuration items from database
serverGerbera # [   27.493221] gerbera[918]: 2021-05-28 00:02:00   info: Loading autoscan location: /var/lib/gerbera/pictures; recursive: false, last_modified: unset
serverGerbera: must succeed: curl --fail http://serverGerbera:49152/
serverGerbera # [   27.500563] gerbera[918]: 2021-05-28 00:02:00   info: Loading autoscan location: /var/lib/gerbera/audio; recursive: true, last_modified: unset
serverGerbera # [   27.506278] gerbera[918]: 2021-05-28 00:02:01   info: Initialising UPnP with interface: eth1, port: 49152
serverGerbera # [   27.511506] gerbera[918]: 2021-05-28 00:02:01   info: IPv4: Server bound to: 192.168.1.2:49152
serverGerbera # [   27.514627] gerbera[918]: 2021-05-28 00:02:01   info: IPv6: Server bound to: fe80::5054:ff:fe12:102:49152
serverGerbera # [   27.516835] gerbera[918]: 2021-05-28 00:02:01   info: IPv6 ULA/GLA: Server bound to: :49153
serverGerbera # curl: (52) Empty reply from server
serverGerbera: output:
error:
Traceback (most recent call last):
  File "/nix/store/mi8ay9gb6kv346mk3sz4gwj6r2nkvbkl-nixos-test-driver/bin/.nixos-test-driver-wrapped", line 927, in run_tests
    exec(tests, globals())
  File "<string>", line 1, in <module>
  File "<string>", line 23, in <module>
  File "/nix/store/mi8ay9gb6kv346mk3sz4gwj6r2nkvbkl-nixos-test-driver/bin/.nixos-test-driver-wrapped", line 466, in succeed
    raise Exception(
Exception: command `curl --fail http://serverGerbera:49152/` failed (exit code 52)
cleaning up
killing client (pid 10)
killing serverGerbera (pid 21)
(0.00 seconds)
builder for '/nix/store/58cp2gzb9r8b9k7sqrv234zvp862plwr-vm-test-run-mediatomb.drv' failed with exit code 1
error: build of '/nix/store/58cp2gzb9r8b9k7sqrv234zvp862plwr-vm-test-run-mediatomb.drv' failed
```

With gerbera understandibly being the new default, this is pretty bad.

File: nixos/doc/manual/release-notes/rl-2105.xml [hunk 1]
```diff
@@ -242,6 +242,24 @@
       <literal>/var/lib/powerdns</literal> to <literal>/run/pdns</literal>.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       The <link linked="opt-services.mediatomb">mediatomb service</link> is
```
[the remark this answers is not in the document]
Comment (schmittlauch) [reply nixos/doc/manual/release-notes/rl-2105.xml:247]: `opt-services.mediatomb` is no reference target, only its sub-options are.

**Review by schmittlauch — CHANGES_REQUESTED** [review]
After cleaning up all the proposed typos and misconceptions, I can confirm that the manual/ release notes finally build successfully.

**Comment by schmittlauch** [comment]
superseded by #124768

**Pull request closed without merging**